phoenix-core/src/it/java/org/apache/phoenix/end2end/OnDuplicateKeyIT.java

```diff
@@ -720,6 +726,402 @@ public void testMultiplePartialUpdatesInSameBatch() throws Exception {
         }
     }
 
+    @Test
+    public void testColumnsTimestampUpdateWithAllCombinations() throws Exception {
```

Comment:
`testColumnsTimestampUpdateWithAllCombinations()` and `testColumnsTimestampUpdateWithIntentionalUpdate()` are the only tests that don't need to be ignored because we are not asserting return status?

Reply:
We are asserting return status, but since the return values are 1 in these two test cases, they will always pass even though hbase change is not included.